x-pack/plugins/rollup/public/crud_app/services/routing.js

```diff
@@ -10,24 +10,34 @@
 
 import { createLocation } from 'history';
 
+let _userHasLeftApp = false;
+
+export function userHasLeftApp(flag) {
```

Comment:
this seems weird to have the same function be a getter and a setter.  Maybe we can split this into a setter/getter pair? (setUserHasLeftApp/userHasLeftApp).  This is where Ruby is nice as you could have a userHasLeftApp? method.

Reply:
Done!